external-providers/yq-external-provider/pkg/generic/dependency.go

```diff
@@ -0,0 +1,39 @@
+package generic
```

Comment:
I would change the package name from `generic` to something like `yq_provider` to avoid conflating this with the generic-external-provider